sql/report_queries/fund/fund_expenditure_summary.sql

```diff
@@ -0,0 +1,93 @@
+/*
+PURPOSE
+This report shows fund expenditures as the sum of the transaction amount by purchase order line grouped by fund and expense class. 
+Fund expenditures can be viewed by fund name, fund type, fund code, purchase order line format, transaction type, expense class, and fiscal year.
+
+MAIN TABLES INCLUDED
+finance_funds
+finance_fiscal_years
+finance_transaction_invoices (derived)
+finance_group_fund_fiscal_years
+po_lines
+
+AGGREGATION
+fund_id, fund_name, fund_type_name, fund_code, fund_description, transaction_type, expense_class, fiscal_year_code 
+
+FILTERS FOR USERS TO SELECT 
+invoice_payment_start and invoice_payment_end, fiscal_year, fund_code, fund_type, purchase_order_line_format, transaction_type
+ 
+*/
+
+/* Enter parameters or leave blank to include all for each filter below. For dates, you may use the invoice payment start and end dates, fiscal year, or both. */
+WITH parameters AS (
+    SELECT
+        /* enter invoice payment start date and end date in YYYY-MM-DD format */
+    	--'2000-01-01' :: DATE AS start_date,
+        --'2021-01-01' :: DATE AS end_date,
+        /* enter FY## where #### is the 4 digits of the fiscal year code */
+        ''::VARCHAR AS fiscal_year_filter,
+        /* enter one or more fund codes separated by commas, as in 'math, music' */
+        ''::VARCHAR AS fund_code_filter,
+        /* enter one or more fund types separated by commas, as in 'restricted, unrestricted' */
+        ''::VARCHAR AS fund_type_filter,
+        /* enter purchase order line order format as 'Physical Resource, Electronic Resource, P/E Mix, or Other' */
+        ''::VARCHAR AS order_format_filter,
+        /* enter one or more expense class names separated by commas, as in 'Electronic, Print'*/
+        ''::VARCHAR AS expense_class_filter,
+        /* enter transaction type format as 'payment','pending payment', or 'payment,pending payment' */
+        ''::VARCHAR AS transaction_type_filter
+)
+
+SELECT
+	fitrin.transaction_from_fund_id AS fund_id,
+	fitrin.transaction_from_fund_name AS fund_name,
+	finfuntyp.name AS fund_type_name,
+	fitrin.transaction_from_fund_code AS fund_code,
```

Comment:
May I suggest to change these 4 lines into:
```
	finfun.id AS fund_id,
	finfun.name  AS fund_name,
	finfuntyp.name AS fund_type_name,
	finfun.code  AS fund_code,
```
We should take these values from finance_funds because when we have no relating transactions for a fund this values will stay empty.